### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by kaitcham**
For this branch me and Evans created the contact form by :

- Adding it's header and description
- We also added the form fields for name, email and comment section
- I concluded by adding the screenshot of the created branch in the README file.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -257,3 +265,59 @@ section {
     flex-direction: column;
   }
 }
+
+.section-form {
+  background: #6070ff;
+  border-top-left-radius: 100px;
+  margin-top: 60px;
```
Comment (BenMukebo) [inline style.css:272]: - [ ] Please, Consider making your design to match the Figma design, at this point, the background is supposed to be an image, not a color, kindly fix that.

File: node_modules/.bin/node [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
```
Comment (BenMukebo) [inline node_modules/.bin/node:1]: - [ OPITIONAL]  for making your repository more professional and clean, please always remember to ignore the node_modules and to not add it to the Github repository, I strongly recommend you remove it. 
node

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi **@kaitcham,**

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!

### To highlight 💪🏻

- No linter errors. ✔️ 
- Github flow was correctly used.  ✔️   
- Good commit messages ✔️ 
- Professional README ✔️ 
- Descriptive README file.  ✔️ 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏 💻  

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Wes-Isaac — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**